src/Samples/DIPS.Xamarin.UI.Samples/Controls/FloatingActionMenu/FloatingActionMenuPage.xaml

```diff
@@ -26,21 +26,23 @@
                                               CloseOnOverlayTapped="{Binding Source={x:Reference shouldCloseOnTappedOverlayCheckbox}, Path=IsChecked}"
                                               IsVisible="{Binding Source={x:Reference toggleVisibilityCheckbox}, Path=IsChecked}">
                 <dxui:MenuButton BackgroundColor="LightSeaGreen"
-                                 Text="A"
+                                 Text="HEI"

```

Comment:
Update samples with values that makes more sense :P Maybe the buttons text could have something like "Home", "About", "Notifications" etc.